### ISSUE TITLE
Can't open /tmp/udocker/init_script.sh

### ISSUE THREAD

**Issue by gmolto** [grycap/scar#144]
Attempting to reproduce the `imagemagick` use case:
```sh
scar init -s examples/imagemagick/grayify-image.sh -n lambda-imagemagick -es scar-test grycap/imagemagick
```
followed by an upload to S3 yields the following error (shown in CloudWatch Logs):

```sh
Can't open /tmp/udocker/init_script.sh
```

**Comment by gmolto** [comment]
This affects to all the use-cases where a shell-script is defined to be executed upon invocation of the Lambda function. A simple test also fails:

```sh
scar init -s test/test-env.sh -n scar-ubuntu-16-04 ubuntu:16.04
```

```sh
scar run scar-ubuntu-16.04
```

Yields the same error.

**Comment by alpegon** [comment]
Fixed in PR #151
